BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx

```diff
@@ -27,7 +28,10 @@ class CancelBuildButton extends Component {
     this.setState({
       cancelling: true,
       showModal: false
-    });    
+    });
+
+    const parentPath = window.location.pathname.slice(0, window.location.pathname.lastIndexOf('/'));
```

Comment:
use [browserHistory](https://github.com/reactjs/react-router/blob/master/docs/guides/Histories.md)

```
import { browserHistory } from 'react-router'
```

then you can trigger - 

```
browserHistory.goBack();
```